pkg/config/types/default_config.go

```diff
@@ -81,9 +81,11 @@ var Default = Bacalhau{
 				Resources: ResourcesConfig{
 					CPU:    "500m",
 					Memory: "1Gb",
+					Disk:   "1Gb",
```

Comment:
Current defaults are:
```
DefaultJobResourceLimits: models.ResourcesConfig{
			CPU:    "500m",
			Memory: "1Gb",
			Disk:   "",
			GPU:    "",
		},
```

Reply:
You are right, these are the defaults before applying this change.
But I thought we just agreed that any job sent from the orchestrator to the compute node without resources populated is invalid: https://github.com/bacalhau-project/bacalhau/pull/4333#discussion_r1744568347. This change ensures that all fields of the job resources are populated with default values when the user doesn't provide any.

Reply:
populating jobs with default resources is not a new requirement. It is the existing behaviour and the goal is the maintain this behaviour as we change our config setup. Lets keep using the existing values